doc/api/fs.md

```diff
@@ -3072,6 +3072,40 @@ extended part is filled with null bytes (`'\0'`):
 
 If `len` is negative then `0` will be used.
 
+descriptor (`fd`). The `length` argument indicates
```

Comment:
I don't understand why this starts with `descriptor (fd)`.

Reply:
I sent an update to make this place better, thank you for your suggestion